reservation_units/models.py

```diff
@@ -280,6 +281,25 @@ class ReservationUnit(models.Model):
         default=0,
         help_text="Maximum price of the reservation unit",
     )
+    TAX_PERCENTAGE_0 = Decimal("0.00")
```

Comment:
I guess there's been talks that tax percentages can be enums? Possible tax changes needs code changes, but that doesn't happen that often luckily, but for maintainability it would be nice that it would be own model (that can be edited in django admin).

Reply:
Good idea -- the current is based on Respa which has it as an enum like this. But like you suggested, having it as a separate model would be better.

Reply:
`TaxPercentage` now has a new model that can be edited via Django admin.

Also, the tax percentages have to be given as PKs for reservation units.